plugins/woocommerce/tests/php/src/Checkout/Shortcode_Checkout_Hook_Test.php

```
@@ -0,0 +1,339 @@
+		$calls = [];
+
+		// Register callbacks with different priorities
+		add_action( 'woocommerce_review_order_after_cart_item_meta', function( $product, $cart_item, $cart_item_key ) use ( &$calls ) {
+			$calls[] = 'first';
+		}, 5, 3 );
+
+		add_action( 'woocommerce_review_order_after_cart_item_meta', function( $product, $cart_item, $cart_item_key ) use ( &$calls ) {
+			$calls[] = 'second';
+		}, 10, 3 );
+
+		add_action( 'woocommerce_review_order_after_cart_item_meta', function( $product, $cart_item, $cart_item_key ) use ( &$calls ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$product'. (undefined)

(UnusedFormalParameter)